js-packages/@prestojs/rest/src/paginationMiddleware.ts

```diff
@@ -4,6 +4,10 @@ import {
     PaginatorInterface,
     PaginatorInterfaceClass,
 } from '@prestojs/util';
+import cloneDeep from 'lodash/cloneDeep';
+import get from 'lodash/get';
+
```

Comment:
I'm surprised the sorting plugin didn't fix the space. Is it configured for Presto?